### ISSUE TITLE
coffea.hist.plot.clopper_pearson_interval gives wrong result for numerator==denominator

### ISSUE THREAD

**Issue by AndreasAlbert** [scikit-hep/coffea#138]
**Describe the bug**
If numerator == denominator, clopper_pearson_interval always returns [1,1] as the corresponding interval.

**To Reproduce**
```python
from coffea.hist.plot import clopper_pearson_interval
clopper_pearson_interval(10,10,0.95)
# Result: array([1., 1.])

````
**Expected behavior**
I expected to get array([0.69150289, 1.])

**Proposed fix**
This is trivially fixed by changing [this line](https://github.com/CoffeaTeam/coffea/blob/2ae59e8c75fa8e5af9a245e27bd7aa363a4f116b/coffea/hist/plot.py#L66) from
```python
interval[:, num == denom] = 1.
```
to 
```python
interval[1, num == denom] = 1.
```

If you agree, I'd be happy to make a PR.

**Comment by lgray** [comment]
@AndreasAlbert thanks for the catch. Please make a PR. 

If you could add a few tests to check this problem and maybe some others in your PR to make sure this error calculator is covered for the future we'd really appreciate it!

**Comment by lgray** [comment]
This fix is now available in 0.6.4 in pypi.
